simglucose/simulation/env.py

```diff
@@ -34,7 +36,58 @@ def risk_diff(BG_last_hour):
 
 
 class T1DSimEnv(object):
+    '''
+    Observation:
+        Type: Box(4)
```

Comment:
Observation only contains BG. So it is a Box(1), with min 70 and max 350.

Reply:
In fact the observation is the blood glucose value measured by CGM (continuous glucose monitor) sensor. BG is usually used to refer to the true blood glucose level.